addons/mail/static/src/new/thread/message.xml

```diff
@@ -89,11 +89,17 @@
                         </div>
                     </div>
                 </div>
-                <div t-else="" class="o-mail-message-editable-content flex-grow-1" t-att-class="{
-                    'p-1': message.isNote,
-                    'o-mail-message-body text-break mb-0 rounded-end-3 rounded-bottom-3 border p-3': !message.isNote,
-                }">
-                    <Composer autofocus="true" composer="message.composer" onDiscardCallback.bind="exitEditMode" onPostCallback.bind="exitEditMode" mode="'compact'"/>
+                <div t-else="" class="position-relative">
+                    <div class="o-mail-message-bubble border rounded-end-3 rounded-bottom-3 position-absolute top-0 start-0 w-100 h-100" t-att-class="{
+                            'bg-info-light border-info': !message.isAuthor,
+                            'bg-success-light border-success opacity-25': message.isAuthor,
+                        }"/>
+                    <div class="o-mail-message-editable-content position-relative flex-grow-1" t-att-class="{
```

Comment:
`o-mail-message-editable-content` classname was in bubble `div`. Bubble is only for styling purposes, whereas this class had semantics that is handy for test assertions.
Tests were failing because composer was not child.
So I just moved in the proper `div`.